src/Akka.Persistence.Azure/Query/Messages.cs

```diff
@@ -172,6 +172,16 @@ public ReplayTaggedMessages(long fromOffset, long toOffset, long max, string tag
         }
     }
 
+    public sealed class ReplayTaggedMessageSuccess
+    {
+        public ReplayTaggedMessageSuccess(bool completed)
+        {
+            Completed = completed;
+        }
+
+        public bool Completed { get; }
+    }
```

Comment:
New message to signal that the database have returned all available rows and the operation has been completed.